Source/WTF/wtf/PlatformWPE.cmake

```diff
@@ -42,6 +42,7 @@ list(APPEND WTF_PUBLIC_HEADERS
     glib/RunLoopSourcePriority.h
     glib/Sandbox.h
     glib/SocketConnection.h
+    glib/SpanHelpers.h
```

Comment:
Can we call this GBytesSPI.h or GBytesExtras.h? We're adding a lot of these helpers and they'll conflict if we give them generic names.

Reply:
I am planning to add more conversion helpers in the future, some of them not related to `GBytes`. How about `GSpanConversion.h` or `GSpanExtras.h`?

Reply:
I went with `GSpanExtras.h` in the end.